docs/manual_doc/Statement_Syntax/Data_Definition_Statements/Drop_Tag_Syntax.md

```diff
@@ -0,0 +1,11 @@
+```
+DROP TAG tag_name
+```
+You must have the DROP privilege for the tag.
+>Be careful with this statement
+
+Since a vertex can have either only one tag (types) or multiple tags (types). 
+In the former case, such a vertex can NOT be accessable after the statement --- which may result in DANGLING edges those are connecting with such vertex.
+In the later case, such a vertex is still accessable. But all the properties defined by this dropped tag are not accessable.
```

Comment:
>is still accessable
Here accessible should be better. `accessable` only exists on the web but not the dictionary.